sources/web/build.sh

```diff
@@ -29,12 +29,22 @@ WEB_DIR=$BUILD_DIR/web/nb
 
 mkdir -p $WEB_DIR
 
+# Experimental UI build step
+cd datalab/polymer
+bower install
+tsc
+polymer build
+rsync -avpq ./build/experimental/ ../static/experimental
```

Comment:
Actually this one is necessary, and the one below is the redundant one. I'll remove that.